fem/pgridfunc.hpp

```diff
@@ -25,7 +25,17 @@
 namespace mfem
 {
 
-/// @brief Compute a global Lp norm from the local Lp norms computed by each processor
+/// @brief Compute a global Lp norm from the local Lp norms computed by each
+///        processor
+///
+/// @param[in] p         Real value indicating the exponent of the $L^p$ norm.
+///                      To avoid domain errors p should have a positive
+///                      value, either finite or infinite.
+/// @param[in] loc_norm  Local $L^p$ norm as computed separately on each
+///                      processor.
+/// @param[in] comm      MPI Communicator
+///
+/// @return              Global $L^p$ norm distributed to each processor
```

Comment:
`distributed to each processor` -> `reduced over all processors`?

Reply:
Maybe. I was trying to not use MPI specific jargon (and keep it short) but perhaps the comment is still unclear

Reply:
The point I wanted to make was that the same value is returned on every processor. I'm not sure that's clear.

Reply:
`Global $L^p$ norm, returned on every processor`?